### PR TITLE
Changed ES7 to ES2016 as per comment

### PR DESCRIPTION
**Pull request by hitautodestruct**
Changed the ES7 nickname to ES2016 as per a comment request. Also added `Vue.component` so syntax highlighter will pick up on the code.

### CODE REVIEW

File: src/v2/guide/reactivity.md [hunk 1]
```diff
@@ -119,10 +119,11 @@ Vue.component('example', {
 })
 ```
 
-Since `$nextTick()` returns a promise, you can achieve the same as the above using the new [ES7 async/await](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) syntax:
+Since `$nextTick()` returns a promise, you can achieve the same as the above using the new [ES2016 async/await](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) syntax:
 
 ```
-...
+Vue.component('example', {
```
Comment (phanan) [inline src/v2/guide/reactivity.md]: Hmm, can't you just use ```js for code syntax highlighting?

Comment (hitautodestruct) [reply src/v2/guide/reactivity.md]: Don't know.. can you?
I am not familiar with the syntax highlighter the vuejs docs use.

Comment (phanan) [reply src/v2/guide/reactivity.md]: Try it ;)

**Comment by phanan** [comment]
Thanks!

**Pull request merged**